tcms/rpc/tests/test_testcase.py

```diff
@@ -64,6 +137,11 @@ def _fixture_setup(self):
                 plan=[self.plan])
             self.cases.append(test_case)
 
+    def test_filter_query_none(self):
+        result = self.rpc_client.TestCase.filter()
+        self.assertIsNotNone(result)
+        self.assertEqual(len(result), 10)
```

Comment:
Convert the second assertion b/c it will break the moment there are more temporary objects into the database.  Check that it is not an empty list, that would be sufficient.